scipy/signal/filter_design.py

```diff
@@ -252,8 +254,8 @@ def freqs_zpk(z, p, k, worN=None):
     return w, h
 
 
-def freqz(b, a=1, worN=None, whole=False, plot=None):
-    """
+def freqz(b, a=1, worN=512, whole=False, plot=None, axis=0):
```

Comment:
... in other words, I think that taking this position as a reason to use `None` instead of `512` leads us to have to something like "there should never be explicit non-None defaults", which doesn't make sense to me.